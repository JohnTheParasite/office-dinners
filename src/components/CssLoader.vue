<template>
  <div class="lds-dual-ring"></div>
</template>

<script>
export default {
  name: "CssLoader"
}
</script>

<style lang="scss" scoped>
@import "../scss/components/color";

.lds-dual-ring {
  display: inline-block;
  width: 0.8rem;
  height: 0.8rem;
}

.lds-dual-ring:after {
  content: " ";
  display: block;
  width: 0.8rem;
  height: 0.8rem;
  border-radius: 50%;
  border: 3px solid;
  border-color: $white transparent $white transparent;
  animation: lds-dual-ring 1.5s linear infinite;
}

@keyframes lds-dual-ring {
  0% {
    transform: rotate(0deg);
  }
  100% {
    transform: rotate(360deg);
  }
}
</style>
