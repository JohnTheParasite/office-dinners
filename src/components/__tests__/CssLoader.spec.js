import CssLoader from "@/components/CssLoader"
import { shallowMount } from "@vue/test-utils"

describe("Render component", () => {
  it("should render loader", () => {
    let component = shallowMount(CssLoader)
    expect(component).toMatchSnapshot()
  })
})
