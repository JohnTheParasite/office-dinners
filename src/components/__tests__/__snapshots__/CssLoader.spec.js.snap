// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render component should render loader 1`] = `<div class="lds-dual-ring"></div>`;
